src/commands/force/org/list.ts

```diff
@@ -75,12 +75,13 @@ export class OrgListCommand extends SfdxCommand {
         ? groupedSortedOrgs.scratchOrgs
         : groupedSortedOrgs.scratchOrgs.filter(identifyActiveOrgByStatus),
     };
-
+    this.ux.warn(messages.getMessage('scratchOrgRemoveConnectedStatusProperty'));
```

Comment:
I wonder if we should just add only if `--json` is used. Pro: only applicable to people who care. Con: it is easier to miss.

Reply:
I thought about it.  Open to opinions.